assets/src/dashboard/templates/data/beauty.js

```diff
@@ -4384,12 +3424,12 @@ export default function (imageBaseUrl) {
             id: 'c05c1df5-86f8-4455-8ef1-c2556787d6b0',
           },
         ],
-        backgroundElementId: 'd07dfc60-f3b6-4295-b687-012dd2cfdf8c',
         backgroundOverlay: 'none',
         type: 'page',
         id: 'edf299b1-e35a-4049-9e66-a08dcee8b936',
       },
     ],
+    version: 19,
```

Comment:
A big nit, but if it was at all possible to stick the version up top, that'd be really nice.

Reply:
Fixed!